hack/e2e.sh

```diff
@@ -102,7 +102,7 @@ function test_elasticsearchcluster() {
     # Create and delete an ElasticSearchCluster
     if ! kubectl create \
             --namespace "${USER_NAMESPACE}" \
-            --filename "${ROOT_DIR}/docs/quick-start/es-cluster-demo.yaml"; then
+            --filename "${SCRIPT_DIR}/testdata/es-cluster-test.yaml"; then
```

Comment:
❓ I'm a bit sad that we'll no longer be testing the documented resource. The documentation will now diverge from the test data and maybe become invalid.

Reply:
Yep it will - if we want to test docs though, I think we should have separate tests. Our test fixtures are going to necessarily diverge from docs, due to disabling persistence, creating single node clusters, etc.

Our tests should be 'narrow' and test individual parts of Navigator's behaviour, so we can end out with an ElasticsearchCluster manifest that is specific to testing say, persistence. Another that's specific to testing a custom port is set, and so on. If all of these checks pass, then we know that the documented examples also pass (because we cover the full set of functionality described in the manifest).